cucumber/screenshots/README.txt

```diff
@@ -0,0 +1 @@
+SAVED SCREENSHOTS APPEAR IN THIS FOLDER
```

Comment:
There is an opportunity to keep empty folder in git repository. Could you please use it instead of this readme file? https://www.theserverside.com/blog/Coffee-Talk-Java-News-Stories-and-Opinions/gitkeep-push-empty-folders-git-commit

Reply:
done